src/odemis/driver/xt_client.py

```diff
@@ -2693,9 +2696,9 @@ def _acquire(self):
                             raise
 
                     # Acquire the image
-                    # =================
                     image = self.parent.get_latest_image(self.parent._scanner.channel)
 
+                    # Add metadata to the image
```

Comment:
Normally we copy the metadata from just after the acquisition started. This way, if the settings are changed during acquisition (eg, dwell time), it's not the new value (which was not yet in use) that's recorded, but properly the original value.

Reply:
So this will have to move to above `get_latest_image` and below `scan_image`?

Reply:
Yes, that's typically the best place for such code, as it'll run essentially while waiting for the acquisition to complete.